addons/mail/models/mail_alias.py

```diff
@@ -93,17 +96,51 @@ def _alias_is_ascii(self):
                     alias.alias_name,
                 ))
 
+    def _parse_alias_defaults(self):
+        """Parse alias defaults and returns error if any.
+
+        :return tuple(dict, str): (parsed alias_defaults or None if an error has occurred, error)"""
```

Comment:
This API is not pythonic at all.  Make it return the parsed value or raise an appropriate exception (`ValidationError` looks like the right type to me.)

Reply:
Thanks for your review. 
Actually, it will be discussed with the team and we will probably take another approach (warning the user rather than trying to solve the problem automatically) so the code will change a lot.